### PR TITLE
Adding a description flag for deploy

### PR DESCRIPTION
**Pull request by kushalmalani**
## Description

> Describe the purpose of this pull request.

Adding a description flag for deploy

## 🎟 Issue(s)

Related astronomer/astro#14751

## 🧪 Functional Testing

> List the functional testing steps to confirm this feature or fix.

## 📸 Screenshots

> Add screenshots to illustrate the validity of these changes.

Screenshot 2023-07-27 at 00 14 50
Screenshot 2023-07-27 at 00 15 07

## 📋 Checklist

- [x] Rebased from the main (or release if patching) branch (before testing)
- [x] Ran `make test` before taking out of draft
- [x] Ran `make lint` before taking out of draft
- [x] Added/updated applicable tests
- [x] Tested against [Astro-API](https://github.com/astronomer/astro/) (if necessary).
- [ ] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: astro-client/astro.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ type Client interface {
 	ReportDagDeploymentStatus(input *ReportDagDeploymentStatusInput) (DagDeploymentStatus, error)
 	// Image
 	CreateImage(input CreateImageInput) (*Image, error)
-	DeployImage(input DeployImageInput) (*Image, error)
+	DeployImage(input *DeployImageInput) (*Image, error)
```
Comment (neel-astro) [inline astro-client/astro.go:29]: nit: trying to understand why did we move to the pointer?

**Review by neel-astro — APPROVED** [review]
Functionally looks fine to me, left a comment on moving to the pointer

File: astro-client/astro.go (hunk 1 above)
Comment (kushalmalani) [reply astro-client/astro.go:29]: It was giving me a lint error

`astro-client/astro.go:180:34: hugeParam: input is heavy (88 bytes); consider passing it by pointer (gocritic)`

Comment (neel-astro) [reply astro-client/astro.go:29]: ahh ok

**Review by sunkickr — APPROVED** [review]

**Pull request merged**